hvplot/__init__.py

```diff
@@ -58,23 +58,22 @@
 To report issues go to https://github.com/holoviz/holoviews.
 """
 
-import inspect
 import os
 import sys
-import textwrap
 
 import panel as _pn
 import holoviews as _hv
 
-from holoviews import Store, render  # noqa
+from holoviews import render  # noqa
 
 
-from .converter import HoloViewsConverter
+from .converter import HoloViewsConverter  # noqa
 from .interactive import Interactive
 from .ui import explorer  # noqa
-from .utilities import hvplot_extension, output, save, show  # noqa
+from .util import _PatchHvplotDocstrings, _in_ipython
+from .utilities import help, hvplot_extension, output, save, show  # noqa
```

Comment:
It's a little weird that there are two files with util/utilities, but this was not added to this PR.

Reply:
Yes it's weird :/

Reply:
That's the sort of thing we should clean up for hvPlot 1.0.